style.css

```diff
@@ -32,13 +30,39 @@ body {
   margin: 0 auto;
 }
 
+.text-header {
+  text-align: center;
+}
+
 .first-section {
+  text-align: center;
+  padding: 50px 50px;
+}
+
+.title-first-section {
+  background-color: red;
+  color: #FFFFFF;
+  display: inline-block;
+  padding: 0 8px 0 8px;
+  font-weight: bold;
+}
+
+.first-text {
   background-color: #F4F4F4;
-  border: 4px solid red;
+  padding: 50px 50px;
```

Comment:
You can use the short format for it. When the all four values are the same you can write it like this: padding 50px.

Reply:
Done.